scripts/printReleaseNotesTemplate.ts

```diff
@@ -1,55 +1,553 @@
+    const sanityConfig = {
+      projectId: PROJECT_ID,
+      dataset: DATASET,
+      token,
+      useCdn: false,
+      apiVersion: '2025-03-06',
+    }
 
-const TEMPLATE = `
-# ✨ Highlights
+    try {
+      changelogId = await createSanityChangelogDocuments(
+        sanityConfig,
+        releaseContext,
+        versionDoc,
+        changeDoc,
+        flags.debug,
+      )
+      // Internal Studio URL for editing (shown in console)
+      const studioUrl = `https://admin.sanity.io/structure/docs;changelog;apiChange;${changelogId}`
+      console.log(`✅ Edit changelog entry in Studio: ${studioUrl}`)
+    } catch (error) {
+      console.error('Error creating changelog documents:', error)
+      process.exit(1)
+    }
+  }
+}
 
-## Awesome feature X
+/**
+ * Creates Sanity changelog documents for a release
+ */
+async function createSanityChangelogDocuments(
+  config: any,
+  context: ReleaseContext,
+  versionDoc: any,
+  changeDoc: any,
+  debug?: boolean,
+): Promise<string> {
+  const client = createClient(config)
 
-A few words about the awesome feature X, preferably with screengifs
+  // Extract the changelog ID from the document ID
+  const changelogId = changeDoc._id.replace('drafts.', '')
 
-## Awesome feature Y
+  // Create the documents in Sanity
+  if (debug) {
+    console.log('Version document:', versionDoc)
+    console.log('Changelog document:', changeDoc)
+  }
 
-A few words about the awesome feature Y, preferably with screengifs
+  await client.createOrReplace(versionDoc)
+  console.log(`Created version document: ${versionDoc._id}`)
 
-## Other features
+  await client.createOrReplace(changeDoc)
+  console.log(`Created changelog document: ${changelogId} (as draft)`)
 
-- This is feature is not that important, but worth mentioning anyway
+  return changelogId
+}
 
-# 🐛 Notable bugfixes
-- Fixes 🐞
-- Fixes 🐛
-- Fixes 🦗
+/**
+ * Generates long description for the changelog entry
+ */
+function generateLongDescription(context: ReleaseContext): PortableTextBlock[] {
+  const blocks: PortableTextBlock[] = [
+    // Highlights heading
+    {
+      _key: uuid(),
+      _type: 'block' as const,
+      children: [
+        {
+          _key: uuid(),
+          _type: 'span' as const,
+          marks: [],
+          text: '✨ Highlights',
+        },
+      ],
+      markDefs: [],
+      style: 'h2',
+    },
+  ]
 
-# Install or upgrade Sanity Studio
+  // Add feature commits
+  if (context.features.length > 0) {
+    // Create a list block for features
+    const featureListItems = context.features.map((commit) => {
+      const [, message, hash] = commit.split(' | ')
+      const blockKey = uuid()
+      return {
+        _key: blockKey,
+        _type: 'block' as const,
+        children: [
+          {
+            _key: `${blockKey}0`,
+            _type: 'span' as const,
+            marks: [],
+            text: message.replace(/^feat(\([^)]*\))?:\s*/, ''),
```

Comment:
It would be preferable here if context included the parsed semantic commits instead of raw messages so we don't need to maintain two separate regexes for this